packages/palette/src/elements/TimeRemaining/TimeRemaining.ios.tsx

```diff
@@ -0,0 +1,77 @@
+import { DateTime, Duration } from "luxon"
```

Comment:
Are there significant design differences between the web and ios implementations? Is it possible to share one component rather than duplicate functionality?

Reply:
No there isn't, I should be able to move it into a shared file, they're identical.